### DISCUSSION TITLE
Klipper/Moonraker install

### DISCUSSION THREAD

**Discussion by AZ4Zach** [feelfreelinux/octo4a#167] [Q&A]
Hey all, I've managed to set up an octoprint server on an old phone I have and install/run klipper without issue, but would really like to try running some klipper-specific ui's. Has anyone had any luck installing moonraker/fluidd/mainsail? I've given it a shot myself but haven't had much luck due to inexperience.

**Comment by jlmenut** [comment]
I tried without success too. It seems that there is an incompatibility between the lmdb library and the filesystem
